packages/block-scripts/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "block-scripts",
-  "version": "0.0.13",
+  "version": "0.0.14",
```

Comment:
There is no need to bump package versions – this will be done automatically when #424 is merged

Reply:
Thanks. I'm leaving as is since I will be publishing this version manually